<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtAddUserWindow</class>
 <widget class="QWidget" name="QtAddUserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtAddUserWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QTextEdit" name="PasswordTextEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>password</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QDateEdit" name="BirthdayDateEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="UserTypeComboBox"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QTextEdit" name="UsernameTextEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>username</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTextEdit" name="SurnameTextEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>surname</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTextEdit" name="NameTextEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>name</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="AddButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
